posthog/src/main/java/com/posthog/java/HttpSender.java

```diff
@@ -41,30 +61,80 @@ public HttpSender build() {
     private HttpSender(Builder builder) {
         this.apiKey = builder.apiKey;
         this.host = builder.host;
+        this.maxRetries = builder.maxRetries;
+        this.initialRetryInterval = builder.initialRetryInterval;
         this.client = new OkHttpClient();
     }
 
-    public void send(List<JSONObject> events) {
+    public Boolean send(List<JSONObject> events) {
```

Comment:
(not blocking for this review...)

Reply:
hah yeah that's my inexperience with Java kicking in

Reply:
it's pretty close to being aesthetic... 

But since it can be null, and Java kicks me into "avoid nulls" mode, I find it nicer to use `boolean` as a default.

I believe that it's slightly cheaper to reference a `boolean` than a `Boolean` but can't see us caring in this context